flexmeasures_openweathermap/cli/commands.py

```diff
@@ -146,9 +146,13 @@ def collect_weather_data(location, asset_id, store_in_db, num_cells, method, reg
             "[FLEXMEASURES-OWM] Setting OPENWEATHERMAP_API_KEY not available."
         )
     if asset_id is not None:
-        locations = get_asset_id_location(asset_id)
-    else:
+        locations = [(get_location_by_asset_id(asset_id))]
```

Comment:
I don't think we need the extra brackets, do we?

Reply:
We need it because `locations` is a list. 

```
def save_forecasts_in_db(
    api_key: str,
    locations: List[Tuple[float, float]],
    max_degree_difference_for_nearest_weather_sensor: int = 2,
):
```

Reply:
I meant the () brackets

Reply:
oh, you're right. I put them mistakenly.